plugins/org.eclipse.papyrus.umllight.core/plugin.xml

```diff
@@ -38,4 +38,9 @@
          </Priority>
       </editpolicyProvider>
    </extension>
+    <extension
+         name="UML Subset for UML LIght"
```

Comment:
Mixed case in the name (not sure whether it is presented to the user anywhere?).